literate_notebooks/src-ES/10_transforms.jl

```diff
@@ -17,64 +19,65 @@ gx2 = groupby(x, [:id, :id2])
 
 #-
 
-vcat(gx2...) # back to the original DataFrame
+vcat(gx2...) # de regreso al DataFrame original
 
 #-
 
 x = DataFrame(id = [missing, 5, 1, 3, missing], x = 1:5)
 
 #-
 
-showall(groupby(x, :id)) # by default groups include mising values and are not sorted
+showall(groupby(x, :id)) # por default los grupos incluyen valores faltantes (`missing`) y no están ordenados
 
 #-
 
-showall(groupby(x, :id, sort=true, skipmissing=true)) # but we can change it :)
+showall(groupby(x, :id, sort=true, skipmissing=true)) # pero se puede cambiar :)
 
 #-
 
 x = DataFrame(id=rand('a':'d', 100), v=rand(100));
-by(x, :id, y->mean(y[:v])) # apply a function to each group of a data frame
+by(x, :id, y->mean(y[:v])) # aplica una función a cada grupo de un DataFrame
 
 #-
 
-by(x, :id, y->mean(y[:v]), sort=true) # we can sort the output
+by(x, :id, y->mean(y[:v]), sort=true) # podemos ordenar el output
 
 #-
 
-by(x, :id, y->DataFrame(res=mean(y[:v]))) # this way we can set a name for a column - DataFramesMeta @by is better
+by(x, :id, y->DataFrame(res=mean(y[:v]))) # de esta manera podemos fijar el nombre de una columna - `DataFramesMeta @by` es mejor
 
 #-
 
 x = DataFrame(id=rand('a':'d', 100), x1=rand(100), x2=rand(100))
-aggregate(x, :id, sum) # apply a function over all columns of a data frame in groups given by id
+aggregate(x, :id, sum) # aplica la función sobre todas las columnas de un DataFarme en grupos dados por `:id`
 
 #-
 
-aggregate(x, :id, sum, sort=true) # also can be sorted
+aggregate(x, :id, sum, sort=true) # también se puede ordenar
 
 # *We omit the discussion of of map/combine as I do not find them very useful (better to use by)*
+# *Omitimos la discusión de `map/combine/` pues no las encuentre tan útiles - mejor usar `by`*
 
 x = DataFrame(rand(3, 5))
 
 #-
 
-map(mean, eachcol(x)) # map a function over each column and return a data frame
+map(mean, eachcol(x)) # mapea una función a cada columna y regresa un DataFrame
```

Comment:
regresa o devuelve? the second sounds better to me but maybe it depends on the variety of Spanish

Reply:
Hmmm - devuelve me gusta más. Nice!